### DISCUSSION TITLE
Mastodon "me" link?

### DISCUSSION THREAD

**Discussion by j2xl** [mmistakes/minimal-mistakes#3993] [Q&A]
To claim a URL as my own on Mastodon, I need to set `rel="me"` on a link to my profile. For example: 

```html
<a rel="me" href="https://social.ridetrans.it/@JasonW">Mastodon</a>
```

Using the default theme, when I add Mastodon to my footer links, I instead get: 

```html
<a href="https://social.ridetrans.it/@JasonW" rel="nofollow noopener noreferrer">
  <i class="fab fa-fw fa-mastodon" aria-hidden="true"></i> Mastodon
</a>
```

How can I add this `me` to the footer for Mastodon?

**Comment by merill** [comment]
A workaround I used is to add it to my about me page and had mastodon link to my About Me page.

```
<a rel="me" href="https://infosec.exchange/@merill">Merill</a>
```

See https://github.com/merill/merill.github.io/blob/master/_pages/about.md

See Mastodon profile https://infosec.exchange/@merill

**Comment by j2xl** [comment]
Thank you! I was able to use an invisible `<link rel="me" href="https://myhost/@myname" />` in `_includes/head/custom.html` and now my site is showing as verified.

**Reply by shadow-absorber** [reply]
how did you make it invissible???

**Reply by shadow-absorber** [reply]
never mind... forgot shadow put it into index.html too
